src/index.html

```diff
@@ -4,24 +4,403 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    />
+    <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
     />
-    <title>Catalog</title>
     <link
       rel="preconnect"
-      href="https://fonts.gstatic.com"
+      href="https://fonts.googleapis.com"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
     />
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
-      href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link nav_link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              tablets
```

Comment:
The text 'tablets' should be capitalized to maintain consistency with other navigation items.